identity/conf/routes

```diff
@@ -43,12 +43,3 @@ GET         /complete-consents                      controllers.editprofile.Edit
 # Redirected to Manage
 GET         /privacy/edit                           controllers.editprofile.EditProfileController.redirectToManageEmailPrefs
 GET         /email-prefs                            controllers.editprofile.EditProfileController.redirectToManageEmailPrefs
-GET         /public/edit                            controllers.editprofile.EditProfileController.redirectToManagePublicSettings
-GET         /account/edit                           controllers.editprofile.EditProfileController.redirectToManageAccountSettings
```

Comment:
Can we delete [these objects](https://github.com/guardian/frontend/blob/1091ee7d5dd8a3526285e44ffebfd0dd38cd2da9/identity/app/controllers/editprofile/editprofile.scala#L7-L8) as well?